src/create_safe.test.js

```diff
@@ -65,7 +66,17 @@ describe('Create New Safe', () => {
     await assertElementPresent({ selector: errorMsg.error(errorMsg.required), type: 'Xpath' }, gnosisPage) // check first "required" error in name field
     await clickAndType({ selector: createSafePage.owner_name_field(1), type: 'css' }, gnosisPage, owner2Name) // filling name field
     await assertElementPresent({ selector: errorMsg.error(errorMsg.required), type: 'Xpath' }, gnosisPage) // checking "required" error in address field
-    await clickAndType({ selector: createSafePage.address_field(1), type: 'css' }, gnosisPage, owner2Address)
+    // validate checksum
+    await clickAndType(
+      { selector: createSafePage.address_field(1), type: 'css' },
+      gnosisPage,
+      owner2Address.toUpperCase(),
+    )
+    const OwnerAddressChecksummed = await getInnerText(
+      { selector: createSafePage.address_field(1), type: 'css' },
+      gnosisPage,
+    )
+    expect(OwnerAddressChecksummed).toEqual(owner2Address)
```

Comment:
Didn't write before but here we have the same issue

Reply:
Changed